numpy/core/src/umath/override.c

```diff
@@ -180,18 +179,6 @@ normalize_reduceat_args(PyUFuncObject *ufunc, PyObject *args,
 }
 
 static int
-normalize_outer_args(PyUFuncObject *ufunc, PyObject *args,
-                    PyObject **normal_args, PyObject **normal_kwds)
-{
```

Comment:
Yikes, reduce and accumulate are really not quite the same, despite my comment. What was I thinking...